tutorials/mongoose/deleteMany.md

```diff
@@ -0,0 +1,41 @@
+If you wish to delete several documents in mongoose, use the `deleteMany()` function.
+It takes up to three parameters:
+- **condition**, what a document should contain to be eligible for deletion.
```

Comment:
Make this an ordered list instead of unordered, so `1. ` instead of `- ` at the beginning